### PR TITLE
tidb-binlog: change bidirectional replication file name and image

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

+ Change the file name of the bidirectional replication document.
+ Change the image text to Chinese. 

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs/pull/2030

### CODE REVIEW

**Comment by TomShawn** [comment]
@yikeke @july2993 PTAL

**Comment by TomShawn** [comment]
@yikeke Comments addressed, PTAL again, thanks!

File: reference/tidb-binlog/bidirectional-replication.md [hunk 1]
```diff
@@ -13,15 +14,15 @@ category: reference
 
 > **注意：**
 >
-> 写入两个集群的数据无冲突，即不会在两个集群修改表的同一主键或者唯一索引的行。
+> 集群间双向同步的前提条件是，写入两个集群的数据必须保证无冲突，即在两个集群中，不会同时修改同一张表的同一主键和具有唯一索引的行
```
Comment (yikeke) [inline reference/tidb-binlog/bidirectional-replication.md:17]:
```suggestion
> 集群间双向同步的前提条件是，写入两个集群的数据必须保证无冲突，即在两个集群中，不会同时修改同一张表的同一主键和具有唯一索引的行。
```

File: reference/tidb-binlog/bidirectional-replication.md [hunk 2]
```diff
@@ -13,15 +14,15 @@ category: reference
...
 
 使用场景示例图如下：
 
-![Architect](/media/binlog/bi-repl1.jpg)
+![场景示例图](/media/binlog/bi-repl1.jpg)
```
Comment (yikeke) [inline reference/tidb-binlog/bidirectional-replication.md:21]:
```suggestion
![使用场景示例图](/media/binlog/bi-repl1.jpg)
```

File: reference/tidb-binlog/bidirectional-replication.md [hunk 3]
```diff
@@ -68,7 +69,7 @@ update drainer_repl_mark set val = val + 1 where id = ? && channel_id = ?;
 
 因为 Drainer 无法为 DDL 操作加入标识表，所以采用单向同步的方式来同步 DDL 操作。
 
-集群 A 到集群 B 开启 DDL 同步，集群 B 到集群 A 关闭 DDL 同步。DDL 操作全部在 A 上执行。
+例如集群 A 到集群 B 开启 DDL 同步，集群 B 到集群 A 关闭 DDL 同步。DDL 操作全部在 A 上执行。
```
Comment (yikeke) [inline reference/tidb-binlog/bidirectional-replication.md:72]:
```suggestion
比如集群 A 到集群 B 开启了 DDL 同步，则集群 B 到集群 A 需要关闭 DDL 同步。即 DDL 操作全部在 A 上执行。
```

**Pull request merged**